allennlp/service/servable/models/semantic_role_labeler.py

```diff
@@ -55,3 +39,23 @@ def predict_json(self, inputs: JSONDict) -> JSONDict:
                 })
 
         return results
+
+    @classmethod
+    def from_params(cls, params: Params) -> 'SemanticRoleLabelerServable':
+        tokenizer = Tokenizer.from_params(params.pop("tokenizer"))
+
+        token_indexers = {}
+        token_indexer_params = params.pop('token_indexers')
+        for name, indexer_params in token_indexer_params.items():
+            token_indexers[name] = TokenIndexer.from_params(indexer_params)
+
+        vocab_dir = params.pop('vocab_dir')
+        vocab = Vocabulary.from_files(vocab_dir)
```

Comment:
I tried this on the bidaf one, take a look and see if it's what you had in mind, if so I can do the other ones too